content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -78,95 +71,48 @@ You can request the response format using request headers as follows:
   <caption>Valid request header values for OpenAPI v2 queries</caption>
 </table>
 
-Kubernetes implements an alternative Protobuf based serialization format for the API that is primarily intended for intra-cluster communication, documented in the [design proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/protobuf.md) and the IDL files for each schema are located in the Go packages that define the API objects.
-
-## API versioning
-
-To make it easier to eliminate fields or restructure resource representations, Kubernetes supports
-multiple API versions, each at a different API path, such as `/api/v1` or
-`/apis/rbac.authorization.k8s.io/v1alpha1`.
-
-Versioning is done at the API level rather than at the resource or field level to ensure that the
-API presents a clear, consistent view of system resources and behavior, and to enable controlling
-access to end-of-life and/or experimental APIs.
-
-The JSON and Protobuf serialization schemas follow the same guidelines for schema changes - all descriptions below cover both formats.
+Kubernetes implements an alternative Protobuf based serialization format for
+the API that is primarily intended for intra-cluster communication, documented
```

Comment:
Would you have time to tidy up this sentence on line 74?

The Kubernetes API implements an alternative Protobuf serialization format that is primarily intended
for intra-cluster communication. For more information about this format, see the
[Kubernetes Protobuf serialization design proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/protobuf.md) and the interface definition language (IDL)
files for each schema located in the Go packages that define the API objects.

Reply:
Thanks. Revised as suggested.